conf/messages.en

```diff
@@ -105,3 +105,24 @@ pay-your-dst.making-payment=when making payments.
 pay-your-dst.how-to-pay.1=Find out
 pay-your-dst.how-to-pay.2=how to pay your Digital Services Tax (opens in a new window or tab)
 pay-your-dst.back-to-details=Back to details
+
+# Company Details
+manage-companies-add-company.heading=Add company details
+manage-companies-edit-company.heading=Change company details
+
+companyDetails.title = Company details
```

Comment:
not a show stopper but a preference. The D in details should be capitalised 
title example: http://localhost:9949/auth-login-stub/gg-sign-in

Reply:
@baha-aboukoura 
The page headings follow a pattern, Auth login stub is a stub page, not a production code. If you check all the other page headings they all follow the one similar to **Company details**.
We are using the existing messages from the DST front end for the returns front end.

Reply:
ok thanks